modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -714,10 +713,18 @@ private void handleUpdateMinimalActiveTxTimeCommand(UpdateMinimumActiveTxBeginTi
         }
 
         long minActiveTxBeginTime0 = minActiveTxBeginTime;
+        long timestamp = cmd.timestamp();
 
-        assert minActiveTxBeginTime0 <= cmd.timestamp() : "maxTime=" + minActiveTxBeginTime0 + ", cmdTime=" + cmd.timestamp();
+        assert minActiveTxBeginTime0 <= timestamp : "maxTime=" + minActiveTxBeginTime0 + ", cmdTime=" + timestamp;
 
-        minActiveTxBeginTime = cmd.timestamp();
+        storage.flush(false).whenComplete((r, t) -> {
```

Comment:
Is that intended that only one storage is flushed?

Reply:
Yes, it is. As different table may use different storages, each storage is checked (because this method is going to be called for all partitions).

Reply:
I'm talking about TxStateStorage and not other's table PartitionDataStorage here.